src/presentation/styles/colors.pcss

```diff
@@ -1,3 +1,6 @@
 #app {
   --bg-color-header: #1A1B1C;
+  --color-text-main: #94979A;
```

Comment:
--bg-color-tab && --color-text-main seem to be different different. So maybe --text-color-main or --text-main-color?

Reply:
Fixed in #19